Makefile

```diff
@@ -35,7 +37,8 @@ PIP = bin/pip install --no-index --no-dependencies --find-links $(WHEEL_CACHE) -
 RAWJSFILES = $(shell find $(GUISRC)/app -type f -name '*.js' -not -path "*app/assets/javascripts/*")
 BUILT_RAWJSFILES = $(patsubst $(GUISRC)/app/%, $(GUIBUILD)/app/%, $(RAWJSFILES))
 MIN_JS_FILES = $(patsubst %.js, %-min.js, $(BUILT_RAWJSFILES))
-TEMPLATE_FILES := $(shell find $(GUISRC)/app -type f -name "*.handlebars" -or -name "*.partial")
+OLD_TEMPLATE_FILES := $(shell find $(GUISRC)/app -type f -name "*.handlebars" -or -name "*.partial")
+NEW_TEMPLATE_FILES := $(shell find $(GUISRC)/app/components -type f -name "*.hbs")
```

Comment:
The extension needs to be different from what we previously used so that we can distinguish between new and old, hence the `*.handlebars` to `*.hbs` change. In looking into how other projects handled partials, I found that most still used the `*.hbs` extension. They usually denoted partials by putting them in a `partials` subdirectory or by putting `.partials` in the name, e.g., `foo.partial.hbs`.